modules/enabling-scanner-v4-helm.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-helm_{context}"]
+= Enabling RHACS Scanner V4 when installing with Helm
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable it during installation.
+
+If you are installing {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Create a `values-public.yaml` file that contains the following lines:
++
+[source,yaml]
+----
+scannerV4:
+    disable: false
+----
+. Run the command to install {product-title-short} with customizations, following the instructions in "Install Central using Helm charts with customizations":
++
+[source,terminal]
+----
+helm install -n stackrox --create-namespace \
+  stackrox-central-services rhacs/central-services \
+  -f <path_to_values_public.yaml> -f <path_to_values_private.yaml>
+----
+. If Scanner V4 is successfully enabled, the configuration summary displayed is similar to the following example:
++
+[source,terminal]
+----
+Central Services Configuration Summary:
+
+  Stackrox Version:                            4.7.1
+  Kubernetes Version:                          v1.31.6
+  Kubernetes Namespace:                        stackrox
+  Helm Release Name:                           stackrox-central-services
+  OpenShift Cluster:                           4
+  Scanner V4:                                  enabled
+  Scanner V4 DB Volume:                        PVC (scanner-v4-db)]
+----
+
+For more information, see "Scanner V4 settings for installing {product-title-short} for OpenShift Container Platform by using Helm" or "Scanner V4 settings for installing RHACS for Kubernetes by using Helm".
+
+After you enable Scanner V4 on the cluster where Central is installed and configure your init bundle or cluster registration secret (CRS) to allow communication between Central and the secured cluster, you must enable Scanner V4 on the secured cluster.
```

Comment:
See other comment regarding clarifying (possibly removing) "allow communication between Central and the secured cluster"